omero/sysadmins/server-upgrade.txt

```diff
@@ -51,7 +51,7 @@ actual database user and database name for your installation.
 OMERO.web configuration
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-Check :ref:`unix_omero_web_deployment` or :ref:`windows_omero_web_deployment`,
```

Comment:
The title of this reference is "Deployment (Unix/Linux)"; maybe that also needs rewording to drop the qualifiers?